aspnetcore/fundamentals/servers/index.md

```diff
@@ -50,18 +80,34 @@ For information about how to use Nginx on Linux as a reverse proxy server for Ke
 
 For information about how to use Apache on Linux as a reverse proxy server for Kestrel, see [Using Apache Web Server as a reverse proxy](../../publishing/apache-proxy.md).
 
-## WebListener
+## HttpSys
+
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
 
-If you run your ASP.NET Core app on Windows, WebListener is an alternative that you can use for scenarios where you want to expose your app to the Internet but you can't use IIS. 
+HttpSys is named WebListener in ASP.NET Core 1.x. If you run your ASP.NET Core app on Windows, WebListener is an alternative that you can use for scenarios where you want to expose your app to the Internet but you can't use IIS.
 
-![WebListener](overview/_static/weblistener-to-internet.png)
+![WebListener](weblistener/_static/weblistener-to-internet.png)
 
-WebListener can also be used in place of Kestrel for applications that are exposed only to an internal network, if you need one of its features that Kestrel doesn't support. 
+WebListener can also be used in place of Kestrel for applications that are exposed only to an internal network, if you need WebListener features that Kestrel doesn't support. 
 
-![WebListener](overview/_static/weblistener-to-internal.png)
+![WebListener](weblistener/_static/weblistener-to-internal.png)
 
 For internal network scenarios, Kestrel is generally recommended for best performance, but in some scenarios you might want to use a feature that only WebListener offers. For information about WebListener features, see [WebListener](weblistener.md).
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+If you run your ASP.NET Core app on Windows, HttpSys is an alternative to Kestrel. You can use HttpSys for scenarios where you expose your app to the Internet and you need HttpSys features that Kestrel doesn't support. 
+
+![HttpSys to Internet](httpsys/_static/httpsys-to-internet.png)
+
+HttpSys can also be used for applications that are exposed only to an internal network. 
+
+![HttpSys to internal network](httpsys/_static/httpsys-to-internal.png)
+
+For internal network scenarios, Kestrel is generally recommended for best performance, but in some scenarios you might want to use a feature that only HttpSys offers. For information about HttpSys features, see [HttpSys](httpsys.md).
```

Comment:
Convert comma to semi-colon and add a comma after "scenarios" ... Here's a cut-'n-paste ...

> For internal network scenarios, Kestrel is generally recommended for best performance; but in some scenarios, you might want to use a feature that only HttpSys offers. For information about HttpSys features, see \[HttpSys](httpsys.md).